php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1053,7 +1053,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
                 ['tst_name' => $this->testName]
             );
             $results = $db->pselectWithIndexKey(
-                "SELECT c.examinerID, e.full_name, u.Email
+                "SELECT DISTINCT c.examinerID, e.full_name, u.Email
```

Comment:
@skarya22 I don't think this DISTINCT is necessary since you are querying by CenterID but I don't see any way it would hurt so.. I'm okai with it